Jellyfin.Server/Migrations/Routines/MigrateDisplayPreferencesDb.cs

```diff
@@ -68,9 +68,11 @@ public void Perform()
                 HomeSectionType.SmallLibraryTiles,
                 HomeSectionType.Resume,
                 HomeSectionType.ResumeAudio,
+                HomeSectionType.ResumeBook,
                 HomeSectionType.LiveTv,
                 HomeSectionType.NextUp,
                 HomeSectionType.LatestMedia,
+                HomeSectionType.LatestRelease,
                 HomeSectionType.None,
             };
```

Comment:
The order here got matched with the one in the `DisplayPreferencesController`.

Reply:
You should not edit old migrations

Reply:
> You should not edit old migrations

That was the question here if there are cases the migration is still used. If so it should be talked if a update here makes sense or doesn't.

Reply:
I removed the changes here to keep the PR small.